glide.lock

```diff
@@ -827,4 +850,14 @@ imports:
   - pkg/handler
   - pkg/util
   - pkg/util/proto
+- name: sigs.k8s.io/controller-tools
+  version: 43466124052c1a1aa7d8fd33624b00bc111fb7cf
+  repo: https://github.com/openshift/kubernetes-sigs-controller-tools.git
```

Comment:
This needs to be backed by a proper `glide.yaml` entry to ensure every `make update-deps` picks that and nothing else.